### PR TITLE
PR: Set up and mobile version skeleton

### PR DESCRIPTION
**Pull request by brainskev**
crated html & css file
added a read.me file
added linters
Built  tool bar section in my index.html file
built  a headline section in my index.html file
added css stylesheet

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -2,21 +2,25 @@
...
 <body> 
     <header>
-        <nav> 
-            <a href="#">My Logo</a>
-            <i class="fa-solid fa-bars"></i>
-        </nav>
+        <ul>
+            <li><a href="#">My Logo</a></li>
+            <li><i class="fa-solid fa-bars"></i></li>   
+        </ul>
```
Comment (youmari) [inline index.html]: - [ ] It was great using the `<nav>` tag  so I suggest wrapping your only the hamburger button with the `<nav>` tag cause the hamburger is a part of the navigation of your portfolio

File: index.html [hunk 2]
```diff
@@ -2,21 +2,25 @@
...
     </header>
     <section class="Headline"> 
         <h1>I’m kelvin Glad to see you!</h1>
-        <p>I’m a software developer! I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hestiate to contact me.</p>
+        <p id="one">I’m a software developer! I can help you build a product , feature or website. Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hestiate to contact me.</p>
+        <p id="two">LET'S CONNECT</p>
         <a href="#"></a>
         <div>
```
Comment (youmari) [inline index.html]: - [ ] You should wrap your social media icon in an `ul` tag and each one in a `<li>` tag and the social media should be using the  `<a>` tag  they should be clickable and take you to your social media accounts (if you have them)

- [ ] Also please make sure to export the icons from the Figma design and they should look the same as the one in the Figma design 
- Yours 
image
- Figma 
image

File: index.html [hunk 3]
```diff
@@ -2,21 +2,25 @@
...
 </head> 
 <body> 
     <header>
-        <nav> 
-            <a href="#">My Logo</a>
-            <i class="fa-solid fa-bars"></i>
-        </nav>
+        <ul>
+            <li><a href="#">My Logo</a></li>
+            <li><i class="fa-solid fa-bars"></i></li>
```
Comment (youmari) [inline index.html]: - [ ] Your hamburger button is not positioned correctly, please try to fix that and I suggest aligning the content of the header with `Flex box` in order to make sure everything stays in its place 
image

File: index.html [hunk 4]
```diff
@@ -2,21 +2,25 @@
...
+    <title>Document</title>
 </head> 
 <body> 
     <header>
-        <nav> 
-            <a href="#">My Logo</a>
-            <i class="fa-solid fa-bars"></i>
-        </nav>
+        <ul>
+            <li><a href="#">My Logo</a></li>
```
Comment (youmari) [inline index.html]: - [ ] You should use the right style in order for your logo looks the same as the one in the Figma design I encourage use the style from Figma 
- 
image

File: index.html [hunk 5]
```diff
@@ -2,21 +2,25 @@
...
+            <li><i class="fa-solid fa-bars"></i></li>   
+        </ul>
     </header>
     <section class="Headline"> 
         <h1>I’m kelvin Glad to see you!</h1>
-        <p>I’m a software developer! I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hestiate to contact me.</p>
+        <p id="one">I’m a software developer! I can help you build a product , feature or website. Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hestiate to contact me.</p>
```
Comment (youmari) [inline index.html:22]: - [ ]  Please make sure to use the right font family, font size, color  and font weight  in order to match The Figma design 
- Yours 
image
- Figma
image

File: index.html (hunk 4 above)
Comment (youmari) [inline index.html]: - [ ] There is white space in the right side of the header and headline please try to remove it in order to match the Figma design 
image

**Review by youmari — CHANGES_REQUESTED** [review]
Hi @brainskev , 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!


## Highlights
✔️ Github flow 
✔️ No linter error 
✔️ Great title and short summary 


## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html [hunk 6]
```diff
@@ -2,21 +2,25 @@
...
-    <title>Document</title>
     <script src="https://kit.fontawesome.com/d52fa491d2.js" crossorigin="anonymous"></script>
+    <title>Document</title>
 </head> 
 <body> 
     <header>
-        <nav> 
-            <a href="#">My Logo</a>
-            <i class="fa-solid fa-bars"></i>
-        </nav>
```
Comment (zieeco) [inline index.html:15]: - As required by the previous reviewer, kindly add the hamburger inside the nav tag

File: index.html (hunk 2 above)
Comment (zieeco) [reply index.html]: - As required by the previous reviewer, kindly do the following:
> * [ ]  Please make sure to export the icons from the Figma design and they should look the same as the one in the Figma design
> * [ ] Also, kindly ensure your icons have the correct spacing to match the design as much as you can..

File: index.html (hunk 3 above)
Comment (zieeco) [reply index.html]: > * [ ]  As required by the previous reviewer, Your hamburger button is not positioned correctly, please try to fix that and I suggest aligning the content of the header with `Flex box` in order to make sure everything stays in its place

File: index.html (hunk 5 above)
Comment (zieeco) [reply index.html:22]: > * [ ]  As required by the previous reviewer, pease make sure to use the right font size, color  and font weight  in order to match The Figma design
> * Yours
>   image
> * Figma
>   image

**Review by zieeco — CHANGES_REQUESTED** [review]
Hi @brainskev ,


Great work on making the changes requested by the previous reviewer 👏 
You've done well implementing some of the changes, however, there are some issues that you still need to work on to go to the next project, but you are almost there!


## To highlight 🎯

- No linter errors.✔️
- Nice design. ✔️
- Descriptive PR. ✔️


## Required Changes ♻️
_Check the comments under the review._





## Optional suggestions
- N/A

**_Every comment with the **[OPTIONAL]** prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._**


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**